ee/codegen/src/generators/nodes/api-node.ts

```diff
@@ -27,16 +27,50 @@ export class ApiNode extends BaseSingleFileNode<ApiNodeType, ApiNodeContext> {
       })
     );
 
+    const methodInput = this.nodeInputsByKey.get("method");
+    if (!methodInput) {
+      throw new NodeAttributeGenerationError(
+        'Node input "method" is required but not found.'
+      );
```

Comment:
[blocking] in general, we've been moving to having these all be `"WARNING"`s (see: https://github.com/vellum-ai/vellum-python-sdks/pull/1003) so that we don't kill the _entire_ codegen process.

So, let's:
- add `"WARNING"` params to all of these
- wrap this in `this.workflowContext.addError()`